### PR TITLE
Implement design

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  <link rel="stylesheet" href="./assets/css/main.css" />
+  <link rel="stylesheet" href="./assets/css/all.min.css" />
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo</title>
+</head>
+<body>
+  <div class="site-wrapper">
+    <div class="content">
```
Comment (takrami) [inline index.html]: you can use main tag here.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <h1 class="title">Todo</h1>
+      <ul class="filter-list">
```
Comment (takrami) [inline index.html]: you can put this ul inside nav tag. since it's actually a navigation behaviour.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <li class="filter-item">All tasks</li>
+        <li class="filter-item">completed</li>
+        <li class="filter-item">uncompleted</li>
+      </ul>
+      <ul class="todo-list">
+        <li class="todo-item-add" >
```
Comment (takrami) [inline index.html]: I would suggest put this li outside of the ul (list of todo) and change li to div tag.

File: assets/css/variables.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  /* spacing */
+  --spacing: 8px;
+  --spacing-xsmall: calc(var(--spacing));
+  --spacing-small: calc(var(--spacing) * 2);
+  --spacing-large: calc(var(--spacing) * 3);
+  --spacing-xlarge: calc(var(--spacing) * 4);
+  --spacing-xxlarge: calc(var(--spacing) * 5);
+
+  /* sizing */
+  --sizing: 10px;
```
Comment (takrami) [inline assets/css/variables.css:18]: are they using for font size? if yes please change unit to em from px. keep in mind by changing the unit you need change the values as well. for example not 10em.

File: assets/css/structure.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  padding: 0 var(--spacing-small);
+  margin: 0 auto;
+}
+
+.content {
+  height: 100vh;
```
Comment (takrami) [inline assets/css/structure.css]: this is making issue of space between your elements. remove it from here.

File: assets/css/structure.css [hunk 6]
```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: var(--primary-font);
+  background-color: var(--primary-color);
+  place-items: center;
```
Comment (takrami) [inline assets/css/structure.css]: what is this doing? i think it's extra and you can remove it.

File: assets/css/variables.css [hunk 7]
```diff
@@ -0,0 +1,35 @@
+:root {
+  /* font */
+  --primary-font: "open-sans";
```
Comment (takrami) [inline assets/css/variables.css]: please use another font from google fonts. maybe roboto?

File: assets/css/structure.css [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+input {
+  accent-color:var(--secondary-color);
+  border: none;
+}
+
+.fa-solid {
+  color: var(--quaternary-color);
+  font-size: var(--bold-font);
+  visibility: hidden;
```
Comment (takrami) [inline assets/css/structure.css:70]: please remove this. add icon always should be visible. for edit and delete we should do that.

File: assets/css/structure.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.todo-item-add:hover .fa-solid{
+  visibility: visible;
+}
```
Comment (takrami) [inline assets/css/structure.css]: u can remove this.

File: assets/css/structure.css [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+  padding: 0 var(--spacing-small);
+  margin: 0 auto;
+}
+
+.content {
+  height: 100vh;
+  background-color: var(--sixth-color);
+  display: grid;
+  place-items: center;
+  border: var(--primary-border);
```
Comment (takrami) [inline assets/css/structure.css]: this is not needed. please remove it.

**Review by takrami — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (hedita) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (hedita) [reply index.html]: Done.

File: assets/css/variables.css (hunk 4 above)
Comment (hedita) [reply assets/css/variables.css:18]: I used them for border radius not font size.

File: assets/css/structure.css (hunk 5 above)
Comment (hedita) [reply assets/css/structure.css]: Fixed.

File: assets/css/structure.css (hunk 6 above)
Comment (hedita) [reply assets/css/structure.css]: The total body?what about background color?

File: assets/css/variables.css (hunk 7 above)
Comment (hedita) [reply assets/css/variables.css]: Done.

File: assets/css/structure.css (hunk 8 above)
Comment (hedita) [reply assets/css/structure.css:70]: Done.

File: assets/css/structure.css (hunk 9 above)
Comment (hedita) [reply assets/css/structure.css]: Done.

File: assets/css/structure.css (hunk 10 above)
Comment (hedita) [reply assets/css/structure.css:18]: The border radius?

Comment (hedita) [reply assets/css/structure.css]: Which one do you mean?

File: assets/css/structure.css [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+  grid-template-columns: auto auto auto;
+}
+
+.filter-item {
+  color: var(--primary-color);
+  padding: var(--spacing-xxlarge);
+}
+
+.todo-item-add {
+  width: 350px;
```
Comment (takrami) [inline assets/css/structure.css]: i suggest remove this width from here.

File: assets/css/structure.css [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+  border: var(--secondary-border);
+  border-radius: var(--sizing-xsmall);
+  padding: var(--spacing-small);
+  margin: var(--spacing-small);
+  display: flex;
+  justify-content: space-between;
+}
+
+.todo-item {
+  width: 350px;
```
Comment (takrami) [inline assets/css/structure.css]: i suggest remove this width from here.

Comment (hedita) [reply assets/css/structure.css]: Done.

File: assets/css/structure.css (hunk 11 above)
Comment (hedita) [reply assets/css/structure.css]: Done.

**Review by takrami — APPROVED** [review]

**Pull request merged**